chia/cmds/init_funcs.py

```diff
@@ -99,7 +120,7 @@ def check_keys(new_root: Path, keychain: Optional[Keychain] = None) -> None:
         updated_target = True
```

Comment:
Just want to double check that the 2 lines above this are doing the right thing. Makes sense to me that we want to set the reward address to an observer address, which `all_targets[0]` would be.

Reply:
yes